### PR TITLE
kubeadm: add page

### LINKED ISSUE #2311: page request: kubeadm
Add documentation for `kubeadm`

### PR DESCRIPTION
**Pull request by wbh1**
- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

This will close #2311

### CODE REVIEW

File: pages/common/kubeadm.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+`kubeadm init`
+
+- Bootstrap a Kubernetes worker node and join it to a cluster:
+
+`kubeadm join --token {{token}}`
+
+- Create a new bootstrap token with a TTL of 12 hours:
+
+`kubeadm token create --ttl 12h0m0s`
```
Comment (agnivade) [inline pages/common/kubeadm.md]: 12h0m0s in tokens (`{{}}`)

File: pages/common/kubeadm.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- Check if the Kubernetes cluster is upgradeable and which versions are available:
+
+`kubeadm upgrade plan`
+
+- Upgrade Kubernetes cluster to a specified version:
+
+`kubeadm upgrade apply {{version}}`
+
+- View the kubeadm ConfigMap containing the cluster's configuration:
```
Comment (agnivade) [inline pages/common/kubeadm.md]: Any reason to write ConfigMap in CamelCase ?

File: pages/common/kubeadm.md (hunk 1 above)
Comment (wbh1) [reply pages/common/kubeadm.md]: Should I change the example description, then? To something like "Create a new bootstrap token with a custom TTL"?

File: pages/common/kubeadm.md (hunk 2 above)
Comment (wbh1) [reply pages/common/kubeadm.md]: Yes -- Kubernetes documentation typically writes it that way when referring to a ConfigMap object (e.g. [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) and [here](https://kubernetes.io/docs/tasks/administer-federation/configmap/))

File: pages/common/kubeadm.md (hunk 1 above)
Comment (wbh1) [reply pages/common/kubeadm.md]: This has been updated :)

**Pull request merged**